File: src/04-condicionales/caso/caso.c
#include <stdio.h>

int main(int argc, char *argv[]) {
    int numero;
    char numeroStr[2];

    printf("Introduzca un numero: ");
    fgets(numeroStr, sizeof(numeroStr), stdin);

    switch (numero) {
        case 0:
            printf("Numero neutro con respecto a la suma\n");
            break;
        case 1:
            printf("Numero neutro con respecto a la multiplicacion\n");
            break;
        case 2:
            printf("Base del sistema binario\n");
            break;
        case 3:
            printf("Lados del triangulo\n");
            break;
        case 4:
            printf("Casas de Hogwarts\n");
            break;
        case 5:
            printf("Puntas de una estrella\n");
            break;
        case 6:
            printf("Primer numero perfecto\n");
            break;
        case 7:
            printf("Numero magico\n");
            break;
        default:
            printf("Solo se imprimen resultados para numeros del 0 al 7\n");
    }
}

File: src/04-condicionales/caso/caso.rb
puts "Introduzca un numero"
numero = gets.chomp.to_i

case numero
when 0
  puts "Numero neutro con respecto a la suma"
when 1
  puts "Numero neutro con respecto a la multiplicacion"
when 2
  puts "Base del sistema binario"
when 3
  puts "Lados del triangulo"
when 4
  puts "Casas de Hogwarts"
when 5
  puts "Puntas de una estrella"
when 6
  puts "Primer numero perfecto"
when 7
  puts "Numero magico"
else
  puts "Solo se imprimen resultados para numeros del 0 al 7"
end
